### ISSUE TITLE
Dublin Core metadata

### ISSUE THREAD

**Issue by jasonkarns** [joshbuchea/HEAD#15]
A more comprehensive grammar of attributes for richer metadata information.

http://dublincore.org/documents/dcq-html/

**Comment by dcondrey** [comment]
[https://github.com/dcondrey/HEAD/tree/dc](https://github.com/dcondrey/HEAD/tree/dc)

**Comment by selfthinker** [comment]
A lot of the Dublin Core metadata is [**invalid** in HTML5](http://stackoverflow.com/a/12784883).
So, if/when it gets added, there should at least be a little warning.

**Comment by bmix** [comment]
Are those DC metadata items also invalid in XHTML5?

**Comment by jasonkarns** [comment]
Most of what is said in that stackoverflow link isn't entirely correct. The SO post states that additional link relations _need_ to be added to the registry. Except the spec itself only states _may_. And for the table of listed meta values, the spec itself states the table is `non-normative`.

Also, the DC metadata is listed in the valid meta extensions table from whatwg. (https://wiki.whatwg.org/wiki/MetaExtensions) which is a _wiki_ so adding other meta extensions is essentially open-ended.

In short, dublin core metadata is valid in both HTML5 and XHTML.

**Comment by selfthinker** [comment]
I think it was correct at the time, but things have changed since then. The HTML validator has also been updated since then. All (or most?) Dublin Core metadata is valid HTML5 nowadays.

**Comment by joshbuchea** [comment]
Would be curious to hear a few more opinions on this matter. I haven't researched it enough yet to determine what I think should be included in this project.

@jasonkarns any chance you'd want to open a PR? Apologies if we/I already asked you in the past, I only ask because you've been the biggest advocate for Dublin Core metadata in this project.

**Comment by joshbuchea** [comment]
Added a section for this ✅

**Comment by szepeviktor** [comment]
Thank you.

**Comment by joshbuchea** [comment]
It didn't take long at all, barely over a decade 😅

**Comment by jasonkarns** [comment]
haha, no shade being thrown from me!!!
